core/server/web/api/v2/admin/routes.js

```diff
@@ -154,6 +154,12 @@ module.exports = function apiRoutes() {
     // ## Slack
     router.post('/slack/test', mw.authenticatePrivate, api.http(api.slack.sendTest));
 
+    // ## Sessions
+    // Only need to `getSession` here because this is only route they don't
+    // have to be authenticated for.
+    router.post('/session', auth.session.getSession, auth.session.createSession);
```

Comment:
Should `auth.session.createSession` include the `auth.session.getSession` it's not exactly clear what's happening here :thinking:

Reply:
👍